docs-chef-io/content/inspec/install.md

```diff
@@ -134,16 +139,16 @@ Please choose one of the options below (Press ↑/↓ arrow to move and Enter to
   Skip
 ```
 
-At the first prompt, choose "I already have a license ID". You are then prompted to enter your license key. 
+At the first prompt, choose "I already have a license ID". You are then prompted to enter your license key.
 
 ```bash
 Please choose one of the options below I already have a license ID
-Please enter your license ID:  tmns-34514522-bfe3-4fe7-850c-3fea317c810b-8089
```

Comment:
This is a public-safe dummy value. I wanted to use realistic values. If you feel it is better, documentation-wise, to use a placeholder, I'm amenable.